<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReportWindow</class>
 <widget class="QMainWindow" name="CreateReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreateReportWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_materials">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Материалы:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_causeOfMalfunction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Причина неисправности: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_assistanceProvided">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Оказанная помощь: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_materials">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_causeOfMalfunction">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_assistanceProvided">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_done">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
